src/ui/mock/fakeInitiatives.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+export const fakeInitiatives = {
+  initiatives: [
+    {
+      id: "08a2945c-3e62-4d7a-bf4c-b38b42441bdd",
+      timestampMs: 1592110800000,
+      champions: [],
+      dependencies: [],
+      references: [],
+      contributions: [
+        {
+          key: "28f79984-a7c6-4c54-b603-a90bf5a63e24",
+          title: "Directory Helper functions",
+          timestampMs: 1591246800000,
+        },
+      ],
+      weight: {incomplete: 0, complete: 0},
+      completed: true,
+      title: "Topocount Creates Prototype Node Service",
+    },
+    {
+      id: "6a974075-6094-4cc0-b6b6-0f4e376240a6",
+      timestampMs: 1592542800000,
+      champions: [],
+      dependencies: [],
+      references: [],
+      contributions: [],
+      weight: {incomplete: 0, complete: 0},
+      completed: true,
+      title: "Dandelion Creates webpack Dev Service",
+    },
+    {
+      id: "95f29621-4761-42c9-815a-59e6c5217069",
+      timestampMs: 1592888400000,
+      champions: [],
+      dependencies: [],
+      references: [],
+      contributions: [
```

Comment:
Arent the dependencies, references and contributions supposed to be objects with the following shape:
```ts
export type EdgeSpec = {|
  +urls: $ReadOnlyArray<URL>,
  +entries: $ReadOnlyArray<NodeEntry>,
|};
```

Even if we arent using URLs, we should probably conform to the existing initiative format right? Otherwise they wont be working at all. This should probably be:
```suggestion
      dependencies: {},
      references: {},
      contributions: {
        entries: [
```

Or are we completely abandoning that old version and forcing upgrades to this format?

Reply:
@decentralion decided we should completely rewrite the underlying format to utilize node addresses.